### ISSUE TITLE
tree and adminLTE have some conflicts

### ISSUE THREAD

**Issue by benzunyinzi** [atatanasov/gijgo#296]
Gijgo tree and the newly updated version of adminLTE have some conflicts. 
In the background built by adminLTE, Gijgo tree is used, but it doesn't work.
before:
![image](https://user-images.githubusercontent.com/22743226/37388181-61f6d2d8-279a-11e8-8f6b-7111563c5b90.png)
after:
![image](https://user-images.githubusercontent.com/22743226/37388169-5593336a-279a-11e8-8e7c-bcc02f783eba.png)

**Comment by atatanasov** [comment]
Do you have any javascript errors in the console? Can you provide a sample code that could be in use for replication of the issue.

**Comment by benzunyinzi** [comment]
no.There's has no error in the console.Sorry, I can't provide you with demo. 
If you can, use this plug-in on the basis of updating the latest version of adminLTE, and you may find the same problem.

**Comment by atatanasov** [comment]
Could you try to include gijgo.js after all other javascript files. I guess that this is going to fix the issue.

**Comment by benzunyinzi** [comment]
[图片]
Tree in the latest version of adminLTE also uses similar usage as yours. I think it's conflict here. I don't know if you can fix anything. Oh, yeah, you said that. I tested it. Still no effect.

**Comment by benzunyinzi** [comment]
![_15210107563819](https://user-images.githubusercontent.com/22743226/37462717-debb2e6a-288d-11e8-8254-e7bb2568f2c7.png)

**Comment by atatanasov** [comment]
In this case I think that would be best if you submit an issues in adminLTE and referance this ticket. 
Can you try to exclude your referance of gijgo.js and see how it goes. Do you know what is the version of gijgo that is in use by adminLTE. I'm not able to see where gijgo is in use from their website at adminlte.io

**Comment by benzunyinzi** [comment]
No, no, no, I think you misunderstood. 
AdminLTE doesn't use gijgo. it's just that its internal use of left navigation is similar to yours. 
It's just a coincidence. 
I'm sorry I can't provide you with demo. I'm really sorry.

**Comment by atatanasov** [comment]
I see. AdminLTE and gijgo define jquery function with same called tree, which is making some conflicts. We are going to think about that and improve it in future versions. We may change the name of the tree function to .gijgoTree(). Thanks for the feedback. Unfotunatelly we can't help you in a short term.

**Comment by benzunyinzi** [comment]
Ha ha, okay. Got it. 
You can let me know when it's up to date. Thank you first. 
I figured you'd change your name to .gijgoTree().
After all, this naming is unique.

**Comment by MarioPerini** [comment]
For me it worked to include gijgo after admin-lte. thanks for the hint @atatanasov
